### PR TITLE
[BUD-74] Root Component

### PR DESCRIPTION
**Pull request by lucianojss**
# Root Component

# [BUD-74](https://digitalrig.atlassian.net/browse/BUD-74) Create AppShell Container

## Description

Root app component.
We decided to go over with material UI (at least for now).
Some gravity colours were added to override default Material UI theme.

## Checklist

- [x] Material UI package.
- [x] Material UI sanitize.
- [x] Material UI theme with some gravity colours.

### CODE REVIEW

File: src/styles/theme.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+};
+
+let theme = createMuiTheme({
+  typography: {
+    htmlFontSize: 10, //62,5%
+  },
+  palette: {
+    primary: {
+      main: gravityColors.primary,
+      contrastText: gravityColors.primaryText,
```
Comment (duiliopastorelli) [inline src/styles/theme.ts]: If this refers only to text colour, I believe it should go after line 29

File: src/styles/theme.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    },
+    secondary: {
+      main: gravityColors.secondary,
+      contrastText: gravityColors.secondaryText,
```
Comment (duiliopastorelli) [inline src/styles/theme.ts]: If this refers only to text colour, I believe it should go after line 29

File: src/styles/theme.ts (hunk 1 above)
Comment (lucianojss) [reply src/styles/theme.ts]: It refers to text that is inside of components that use primary palette, it is different from the line 29 text property. You can find more details in material UI documentation.

File: src/styles/theme.ts (hunk 2 above)
Comment (lucianojss) [reply src/styles/theme.ts]: It refers to text that is inside of components that use primary palette, it is different from the line 29 text property. You can find more details in material UI documentation.

**Pull request merged**